angrmanagement/ui/menus/file_menu.py

```diff
@@ -45,6 +52,11 @@ def __init__(self, main_window):
             MenuEntry('E&xit', main_window.quit),
         ])
 
+    def _edit_save(self, **_):
+        enable: bool = not self._project.am_none
+        for i in self._save_entries:
+            (i.enable if enable else i.disable)()
```

Comment:
nit: Simpler to just pass the bool in to a setEnabled function

Reply:
There is no `setEnabled` function; we are editing from the custom class which exposes only these two functions- correct me if I am wrong.

Reply:
https://github.com/angr/angr-management/blob/6c70c966e52ffc99e5d609c3e5261888f4bc5a3c/angrmanagement/ui/menus/menu.py#L5